cr.sh

```diff
@@ -177,6 +177,17 @@ lookup_latest_tag() {
     fi
 }
 
+filter_charts() {
+    while read chart; do
+        local file="$chart/Chart.yaml"
+        if [[ -f "$file" ]]; then
+            echo $chart
+        else
+           echo "Warning: $file is missing, assuming that '$chart' is not a Helm chart. Skipping." 1>&2
```

Comment:
Just one more thing to match the existing style.
```suggestion
           echo "WARNING: $file is missing, assuming that '$chart' is not a Helm chart. Skipping." >&2
```